src/js/component/Proxy.js

```diff
@@ -1,3 +1,13 @@
+import {api} from './Api'
+const proxyHandler = {
+  get: function(target, name){
+        return target[name]
+    }
+}
+const proxiedApi = new Proxy(api, proxyHandler)
+proxiedApi.request()
+proxiedApi.get()
```

Comment:
proxiedApi.request()
proxiedApi.get()

Why do you need it here? It was just an example of usage. With current implementation no one of your requests will not be logged via such proxy. You need to export proxied api and use it as api in all places